console/services/market_app_service.py

```diff
@@ -1341,33 +1356,41 @@ def delete_rainbond_app_version(self, enterprise_id, app_id, version):
             raise e
 
     def get_rainbond_app_and_versions(self, enterprise_id, app_id, page, page_size):
-        have_version = False
         app = rainbond_app_repo.get_rainbond_app_by_app_id(enterprise_id, app_id)
-        app_versions = rainbond_app_repo.get_rainbond_app_versions_by_id(enterprise_id, app_id)
         if not app:
             raise RbdAppNotFound("未找到该应用")
+        app_versions = rainbond_app_repo.get_rainbond_app_version_by_app_ids(enterprise_id, [app_id]).values()
```

Comment:
It is recommended to query only fields other than the template. speed up queries.